### PR TITLE
SecretManager to generate unique AOS password everytime & eks-setup script update

### PR DESCRIPTION
**Pull request by srakshit**
*Issue #, if available:*

*Description of changes:*

- Updated eks-setup script by removing eks cluster creation related commands
- Integrated with SecretManager to generate unique AOS password everytime 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: cf-templates/stack.yaml [hunk 1]
```diff
@@ -1292,7 +1403,6 @@ Resources:
 ## Lambda Get PublicIP Information
   GetEC2PublicIP:
       Type: AWS::Lambda::Function
-      DependsOn: ReverseProxyLaunchConfig
```
Comment (rafael-gumiero) [inline cf-templates/stack.yaml:1295]: Why is this dependency being removed?

File: cf-templates/stack.yaml [hunk 2]
```diff
@@ -66,15 +66,10 @@ Parameters:
...
     Default: "aosadmin"
     Type: String
-
-  MasterUserPassword:
-    Description: Amazon OpenSearch Service - Password
-    Default: "P@ssw0rd!"
-    Type: String
   
   # Reverse Proxy Variables
   ReverseProxyInstanceType:
```
Comment (rafael-gumiero) [inline cf-templates/stack.yaml:75]: Did you test reverse proxy access after this change?

File: scripts/eks-setup.sh [hunk 3]
```diff
@@ -16,7 +16,7 @@ sed -i -e "s/__AOS_ENDPOINT__/$AOS_ENDPOINT/g" observability-with-amazon-opensea
 sed -i -e "s/__AOS_USERNAME__/$AOS_USERNAME/g" observability-with-amazon-opensearch/sample-apps/01-data-preper/kubernetes/data-preper.yaml
 sed -i -e "s/__AOS_PASSWORD__/$AOS_PASSWORD/g" observability-with-amazon-opensearch/sample-apps/01-data-preper/kubernetes/data-preper.yaml
```
Comment (rafael-gumiero) [inline scripts/eks-setup.sh:17]: How are you retrieving these credentials via script?

**Review by rafael-gumiero — CHANGES_REQUESTED** [review]

File: cf-templates/stack.yaml (hunk 1 above)
Comment (srakshit) [reply cf-templates/stack.yaml:1295]: It was not a correct dependency. Rather I have put the dependency on ReverseProxyASG.

File: cf-templates/stack.yaml (hunk 2 above)
Comment (srakshit) [reply cf-templates/stack.yaml:75]: Absolutely!

File: scripts/eks-setup.sh (hunk 3 above)
Comment (srakshit) [reply scripts/eks-setup.sh:17]: The AOS credentials are in Cloudformation output like before.
this command in eks-setup.sh gets the latest AOSPassword - 
`export AOS_PASSWORD=$(aws cloudformation describe-stacks --stack-name ${STACK_NAME} --query "Stacks[0].Outputs[?OutputKey=='AOSDomainPassword'].OutputValue" --output text)`

File: cf-templates/stack.yaml (hunk 1 above)
Comment (srakshit) [reply cf-templates/stack.yaml:1295]:
```
  Lambdatrigger:
     Type: 'Custom::GetEC2PublicIP'
     DependsOn: ReverseProxyASG
     Properties:
       ServiceToken: !GetAtt 'GetEC2PublicIP.Arn'
       GetPublicIP: !Ref GetEC2PublicIP
```

**Review by rafael-gumiero — COMMENTED** [review]
Thank you very much for this important contribution by raising our security bar.

**Review by rafael-gumiero — APPROVED** [review]
LGTM

**Pull request merged**